### ISSUE TITLE
OpenCore boot picker hangs on: Choose the operating system: OK

### ISSUE THREAD

**Issue by ArixElo** [firedevel/Hackintosh-Dell-Latitude-E6520-Catalina#1]
Hello, i do own exact same model but with i5 as on your EFI, but when I try to boot into the installer it gives me the issue which is in the title, tried many things to fix it, nothing helped. 
Can you help me fixing this issue in your free time?

**Comment by firedevel** [comment]
Can you add more details? Like an image or so on?
Have you tried to clear NVRAM or reset and upgrade the BIOS?
Try BIOS version A22 and remember to set SATA mode to AHCI.

**Comment by ArixElo** [comment]
Thank you for responding, yes: I'm trying to boot with a recovery image (as the Dortania guide tells to do), about bios, currently is at A18, because I can't update it with a file from a Dell, or am I stupid enough to not knowing how to update a bios here xD
SATA mode is set to AHCI.
Tried resetting NVRAM twice.

**Comment by ArixElo** [comment]
And it seems like the updating bios didn't helped much :(
I'll try cleaning NVRAM for the third time, but I don't think that would help.

**Comment by firedevel** [comment]
Download: https://www.dell.com/support/home/drivers/driversdetails?driverid=6434j  
There may have two way to update BIOS:  
1. Run it directly in Windows (if it doesn’t work, try to boot windows by legacy boot)
2. F12 and there’s a option, just select the file(.exe) from NTFS/FAT32 partition

**Comment by ArixElo** [comment]
> Download: https://www.dell.com/support/home/drivers/driversdetails?driverid=6434j There may have two way to update BIOS:
> 
> 1. Run it directly in Windows (if it doesn’t work, try to boot windows by legacy boot)
> 2. F12 and there’s a option, just select the file(.exe) from NTFS/FAT32 partition

Let me give an update regarding that: I was able to finally update BIOS with a USB stick prepared in DOS mode, and about macOS: also I was able to boot it, but the method which is on the Dortania's Guide doesn't work for me.
Basically I was forced to download the whole system image with gibMacOS script and make a USB stick with their script. And it booted, so I'm closing the issue since it was from my side.
![Messenger_creation_7116C49A-0BEF-4457-BEC5-70CA8331A49F](https://github.com/user-attachments/assets/6a72da10-2ad8-4e8d-bc18-fde95b8e506a)
